src/proc/formats-converter.h

```diff
@@ -0,0 +1,57 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "processing-blocks-factory.h"
+#include "../core/streaming.h"
```

Comment:
Needed? can we forward declare if yes?

Reply:
Removed the include. Not needed anymore.